### PR TITLE
Bug: fixed total amount calculations

### PR DESCRIPTION
**Pull request by yarpo**
### Change description ###
Fixed calculation of claim total amount.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/app/claims/models/claim.ts [hunk 1]
```diff
@@ -44,9 +44,10 @@ export default class Claim implements Serializable<Claim> {
     const interestRate = this.claimData.interest
     const interestDate = this.claimData.interestDate
     const claimAmount: number = this.claimData.amount
-    const date = interestDate.type === InterestDateType.SUBMISSION ? this.createdAt : interestDate.date
+    const paidFeeAmount: number = this.claimData.paidFeeAmount || 0
+    const date = (interestDate.type === InterestDateType.SUBMISSION) ? this.createdAt : interestDate.date
```
Comment (timja-kainos) [inline src/main/app/claims/models/claim.ts]: no need for brackets here

File: src/main/app/claims/models/claim.ts [hunk 2]
```diff
@@ -44,9 +44,10 @@ export default class Claim implements Serializable<Claim> {
     const interestRate = this.claimData.interest
     const interestDate = this.claimData.interestDate
     const claimAmount: number = this.claimData.amount
-    const date = interestDate.type === InterestDateType.SUBMISSION ? this.createdAt : interestDate.date
+    const paidFeeAmount: number = this.claimData.paidFeeAmount || 0
```
Comment (timja-kainos) [inline src/main/app/claims/models/claim.ts]: not the right fix

File: src/main/app/claims/models/claim.ts (hunk 1 above)
Comment (yarpo) [inline src/main/app/claims/models/claim.ts]: I think it's much more readable

File: src/main/app/claims/models/claim.ts (hunk 2 above)
Comment (yarpo) [inline src/main/app/claims/models/claim.ts]: ok

**Pull request closed without merging**